modules/mono/editor/GodotTools/GodotTools.OpenVisualStudio/Program.cs

```diff
@@ -46,19 +46,15 @@ private static int Main(string[] args)
 
             if (dte == null)
             {
-                // Open a new instance
-                dte = TryVisualStudioLaunch("VisualStudio.DTE.17.0");
+                // Open a new instance, preferring the newest installed VS
+                dte = TryVisualStudioLaunch("VisualStudio.DTE.18.0")   // VS 2026
+                    ?? TryVisualStudioLaunch("VisualStudio.DTE.17.0")  // VS 2022
+                    ?? TryVisualStudioLaunch("VisualStudio.DTE.16.0"); // VS 2019
```

Comment:
VS2019 is no longer supported, so this can be trimmed down. Also, comments should end with a period.
```suggestion
                // Open a new instance, preferring the newest installed VS.
                dte = TryVisualStudioLaunch("VisualStudio.DTE.18.0")   // VS 2026.
                    ?? TryVisualStudioLaunch("VisualStudio.DTE.17.0"); // VS 2022.
```